how-to: add tidb lightning tutorial (#1637)
pingcap/docs: dev/how-to/get-started/tidb-lightning.md

    # The file path in which the engine file is stored.
    import-dir = "/mnt/ssd/data.import/"
    ```

3. Operate `tikv-importer`:

    {{< copyable "shell-regular" >}}

    ```sh
    nohup ./tikv-importer -C tikv-importer.toml > nohup.out &
    ```

### Step 4: Start `tidb-lightning`

1. Upload `bin/tidb-lightning` and `bin/tidb-lightning-ctl` in the installation package to the server where TiDB Lightning is deployed.
2. Upload the [prepared data source](#prepare-full-backup-data) to the server.
3. Configure the parameters for `tidb-lightning` and operate `tidb-lightning`. If you directly use the `nohup` command in the command-line to start the `tidb-lightning` process, the process might exit because of the SIGHUP signal received. It is recommended that you use the `nohup` command in the script. For example:

    {{< copyable "shell-regular" >}}

    ```sh
    #!/bin/bash
    nohup ./tidb-lightning \
                --importer 172.16.31.10:8287 \
                -d /data/my_database/ \
                --tidb-server 172.16.31.2 \
                --tidb-user root \
                --log-file tidb-lightning.log \
            > nohup.out &
    ```

### Step 5: Check data

> After the import is completed, TiDB Lightning exits automatically. If the import is successful, `tidb lightning exit` is displayed in the last line of the log.

Reviewer: ```suggestion
After the import is completed, TiDB Lightning exits automatically. If the import is successful, you can find `tidb lightning exit` in the last line of the log file.
```

> If any error occurs, refer to [TiDB Lightning Troubleshooting](/dev/how-to/troubleshoot/tidb-lightning.md)

Reviewer: ```suggestion
If any error occurs, refer to [TiDB Lightning Troubleshooting](/dev/how-to/troubleshoot/tidb-lightning.md).
```

## Summary

> This tutorial briefly introduces TiDB Lightning, and describes how to deploy a set of TiDB Lightning cluster and imports the full backup data to the TiDB cluster.

Reviewer: ```suggestion
This tutorial briefly introduces what TiDB Lightning is and how to quickly deploy a TiDB Lightning cluster to import full backup data to the TiDB cluster.
```

> For more details about the features and usage of TiDB Lightning, refer to [TiDB Lightning Overview](/dev/reference/tools/tidb-lightning/overview.md).

Reviewer: ```suggestion
For detailed features and usage about TiDB Lightning, refer to [TiDB Lightning Overview](/dev/reference/tools/tidb-lightning/overview.md).
```
